src/lib/server/api/discord.ts

```diff
@@ -73,6 +87,26 @@ export async function dispatchConfessionViaHttp(
         ],
     };
 
+    if (attachment) {
```

Comment:
Similar concern regarding explicit conditions.
```suggestion
    if (attachment !== null) {
```